src/__test__/pages/experiments/[experimentId]/data-exploration/components/differential-expression-tool/DiffExprCompute.test.jsx

```diff
@@ -159,22 +199,14 @@ describe('DiffExprCompute', () => {
       </Provider>,
     );
 
-    let form = component.find(Form);
+    const form = component.find(Form);
 
     // There should be one form.
     expect(form.length).toEqual(1);
 
     // It should have a radio button group at the top
     expect(form.find(Radio.Group).length).toEqual(1);
 
-    // Clicking on the second one should re-render the tool.
-    const { onChange } = form.find(Radio.Group).getElement().props;
-
-    onChange({ target: { value: 'within' } });
-
-    component.update();
-    form = component.find(Form);
-
```

Comment:
if this piece of code is being deleted, how do we trigger a change in the component in order to test if a change is done?

Reply:
Hmm... I might have deleted it to check some stuff. I'll check it again,

Reply:
Previously, the click action changes state, which changes the comparison view. Now, the state is removed and state change is done through redux store. Actions dispatched to `mockStore`, does not change the state. So I removed this code and replaced the values in the store that is used in this test to a state after the radio button is clicked to simulate the clicking. However, this does not reflect the description of the test, which is to test the clicking. 

So, I recovered the code and simulate clicking better. Now the test checks for the action that is fired as the button is clicked, and then tests for the correct UI to be rendered.

Reply:
ok, that sounds good.